### DISCUSSION TITLE
Red Alert ~ Microsoft blocking customizing apps in Windows 11 24H2 "Build: 26100"

### DISCUSSION THREAD

**Discussion by michaelleepow90** [Open-Shell/Open-Shell-Menu#1902] [General]
- Oh, just to be aware, I heard on YouTube that Microsoft will start blocking all customizing apps in 24H2 of Windows 11 Build 26100. I caught the news from the "Windows, Computers & Technology" channel.

**Comment by ge0rdi** [comment]
`Open-Shell` seems to be working fine in build `26100`.

**Reply by taaf** [reply]
Is Open Shell working fine with Windows 11 24H2?

**Reply by ge0rdi** [reply]
As I wrote, it seems to be working fine.

**Reply by taaf** [reply]
OK. Thanks.

**Comment by Ronin-Herobrine** [comment]
I really hope this doesn't happen.

**Reply by Cheopys** [reply]
It's just the kind of tyranny I heard every day when I worked there.

I'll turn off Windows Update forever before I'll go back to the goddamn Smile Start Menu. That was absolutely useless.

**Comment by Aokromes** [comment]
> * Oh, just to be aware, I heard on YouTube that Microsoft will start blocking all customizing apps in 24H2 of Windows 11 Build 26100. I caught the news from the "Windows, Computers & Technology" channel.

the 2 known blocked are the ones who breaks windows starting properly after mayor update.

**Reply by GrogMonster** [reply]
I'm setting up a customer machine with Windows 11 home 24H2 version 26100.1150 - it's the first 24H2 that's come through my workshop. When I try and install openshell I get a popup saying "This app can't run on this device", noting that it may cause security or performance issues. A fair chunk of my business is setting up new computers and transferring data and programs for domestic customers, and of the hundreds (possibly thousands) of machines I've set up since the release of Windows 8 I've had exactly two customers who've declined OpenShell (or ClassicShell) and preferred the Windows 10 start menu (and none who preferred Windows 8 or 11). Every single person who sees the Windows 11 start menu comments about how bad it is. I find it staggering that Microsoft are so willing to ignore public opinion to that extent. Most of my customers are older and don't want to have such a radical change forced onto them - they've used the same start menu for 20 years (since it was introduced with Windows 95) and now to have it change radically 3 times in the last 10 years does nothing but confuse them.

So any tips for getting Openshell running on Win11 24h2 would be very much appreciated.

Interestingly - and at the risk of ranting about 2 things at once - I also can't install Avast Free antivirus, even though I'm getting it from the Windows store!. It tells me to wait a bit and try later. It's been saying that for the last week, so that's not looking promising either.

**Reply by ge0rdi** [reply]
> When I try and install openshell I get a popup saying "This app can't run on this device", noting that it may cause security or performance issues.

Could you attach screenshot of such popup?

What is the machine specification? Couldn't it be some ARM64 machine? We don't support ARM64 atm.

**Reply by GrogMonster** [reply]
It's a Snapdragon X1E80100 3.4GHz, windows 11 home, brand new Microsoft Surface Laptop 15".
![20240723_180600](https://github.com/user-attachments/assets/395eb652-8568-4edd-80ae-dd7bc13aa06b)

**Reply by ge0rdi** [reply]
> It's a Snapdragon ...

That means it is ARM64 device.
We don't support ARM64 😞

**Reply by lotuseaterdotnet** [reply]
> > It's a Snapdragon ...
> 
> That means it is ARM64 device. We don't support ARM64 😞

is there any plan to support ARM64? i am setting up a new a laptop and am devastated by this news...

**Reply by ge0rdi** [reply]
Support for ARM64 was added in [4.4.196](https://github.com/Open-Shell/Open-Shell-Menu/releases/tag/v4.4.196).
Feel free to try :)

**Comment by taaf** [comment]
Please make Open Shell work on Windows forever. I can't use Windows with this terrible start menu that MS forces us to use. I only like Windows 95's start menu.

**Reply by Ibuprophen** [reply]
Unfortunately... To support ARM/ARM64 devices would require allot of work/rework in the software. 

The development team is always open to other developers in providing this specific feature/format. 

I also have an ARM tablet that I'm unable to use this software in. 

I can't personally support this myself, but from what I've seen, [StartAllBack](https://www.startallback.com/) (costs $5) supposed to work on ARM (I think you can also use winget to install it as part of a setup script for customizing new systems).

~Ibuprophen

**Comment by Cheopys** [comment]
The Windows 11 "Start Menu" has no value whatsoever. Instead of a list of installed software, it shows currency conversions, stock prices, and news stories featuring smiling people. Other than bringing up Explorer and searching, there is no way to start installed software.

This rises to the level of sabotage.

**Comment by Rollback-Jockey** [comment]
If it gets to it just uninstall OpenShell, update windows and reinstall OpenShell.Sent from my Galaxy
-------- Original message --------From: Chris Fox ***@***.***> Date: 21/06/2025  18:16  (GMT+00:00) To: Open-Shell/Open-Shell-Menu ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [Open-Shell/Open-Shell-Menu] Red Alert ~ Microsoft blocking
  customizing apps in Windows 11 24H2 "Build: 26100" (Discussion #1902) 
It's just the kind of tyranny I heard every day when I worked there.
I'll turn off Windows Update forever before I'll go back to the goddamn Smile Start Menu. That was absolutely useless.
